cfme/tests/control/test_bugs.py

```diff
@@ -624,3 +632,29 @@ def test_policy_condition_multiple_ors(
             assert state == "Policy simulation successful."
         else:
             assert state == "Policy simulation failed with: false"
+
+
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1740290])
+@pytest.mark.parametrize("page", ["ControlSimulation", "ControlImportExport", "ControlLog"],
+                         ids=["Simulation", "ImportExport", "Log"])
+def test_control_breadcrumbs(appliance, page):
+    """
+    Test to make sure breadcrumbs are visible and properly displayed
+
+    Bugzilla:
+        1740290
+
+    Polarion:
+        assignee: jdupuy
+        caseimportance: high
+        casecomponent: Control
+        initialEstimate: 1/30h
+        startsin: 5.11
+    """
+    view = navigate_to(appliance.server, "ControlExplorer")
```

Comment:
Q: why we are checking every parametrization `ControlExplorer`? 
I think better to make it part of `page`.

Reply:
Talked off line, this is a part of the BZ. We have to navigate to this page first to see the BZ. I will add a comment to help explain this.

Reply:
As per offline discussion this is valid condition as per BZ. 

I like to request you... just comment on this navigation :+1:

Reply:
done.